tests/unit_tests/docmaps/provider_test.py

```diff
@@ -232,18 +211,42 @@ def test_should_populate_second_step_actions_outputs_with_necessary_details(self
             }]
         }]
 
-    def test_should_populate_actions_for_articles_with_evaluations_in_first_step(self):
+    def test_should_populate_inputs_peer_reviewed_step(self):
+        docmaps_item = get_docmap_item_for_query_result_item(
+            DOCMAPS_QUERY_RESULT_ITEM_WITH_EVALUATIONS
+        )
+        peer_reviewed_step = docmaps_item['steps']['_:b2']
+        assert peer_reviewed_step['inputs'] == [{
+            'type': 'preprint',
+            'doi': DOI_1,
+            'url': DOCMAPS_QUERY_RESULT_ITEM_1['preprint_url'],
+        }]
+
+    def test_should_populate_assertions_peer_reviewed_step(self):
+        docmaps_item = get_docmap_item_for_query_result_item(
+            DOCMAPS_QUERY_RESULT_ITEM_WITH_EVALUATIONS
+        )
+        peer_reviewed_step = docmaps_item['steps']['_:b2']
+        assert peer_reviewed_step['assertions'] == [{
+            'item': {
+                'type': 'preprint',
+                'doi': DOI_1,
+                'versionIdentifier': ''
+            },
+            'status': 'peer-reviewed'
+        }]
+
+    def test_should_populate_actions_outputs_with_doi_and_url_peer_reviewed_step(self):
         docmaps_item = get_docmap_item_for_query_result_item(
             DOCMAPS_QUERY_RESULT_ITEM_WITH_EVALUATIONS
         )
-        first_step_key = docmaps_item['first-step']
-        first_step = docmaps_item['steps'][first_step_key]
-        assert first_step['actions'] == [
+        peer_reviewed_step = docmaps_item['steps']['_:b2']
+        assert peer_reviewed_step['actions'] == [
             {
                 'participants': [],
                 'outputs': [
                     {
-                        'type': '',
+                        'type': 'review-article',
```

Comment:
just reading the test it is a bit difficult to see why we are for example expecting this type to be a `review-article`.